test/infrastructure/kind/mapper.go

```diff
@@ -80,6 +80,26 @@ var preBuiltMappings = []Mapping{
 
 	// TODO: Add pre-built images for newer Kind versions on top
 	// Pre-built images for Kind v1.26.
```

Comment:
I think we should have a new section for 1.27

Reply:
Yes. + please change the version number to v0.27. kind is v0.27 not v1.27 (same for the pre-existing commentes below)

Reply:
I'll add a new comment below